packages/cli/.nycrc

```diff
@@ -5,6 +5,16 @@
   "exclude": [
     "src/cmds/preview/depsChecker/**/*",
     "src/cmds/preview/preview.ts",
+    "src/cmds/preview/npmInstallHandler.ts",
+    "src/commonlib/appStudioLoginUserPassword.ts",
+    "src/commonlib/azureLoginCI.ts",
+    "src/commonlib/azureLoginUserPassword.ts",
+    "src/commonlib/codeFlowLogin.ts",
+    "src/commonlib/codeFlowTenantLogin.ts",
+    "src/commonlib/graphLogin.ts",
+    "src/commonlib/graphLoginCI.ts",
+    "src/commonlib/graphLoginUserPassword.ts",
+    "src/commonlib/sharepointLoginUserPassword.ts",
```

Comment:
If wildcard is supported, try to use `*Login.ts`, `*LoginCI.ts`, `*LoginUserPassword.ts`, since more scopes and login methods may be added for new features. (E.g., client credential)

Reply:
Good advice. Updated.